nmsg/config_file.c

```diff
@@ -0,0 +1,387 @@
+/*
```

Comment:
Need a description here or in config_file.h of what style of config file this is.